tests/macros/utils.js

```diff
@@ -183,20 +341,55 @@ function lintHTML(html, fragment = true) {
         });
         return null;
     } catch (error) {
-        const error_message = error.message
+        const error_message = /** @type {Error} */ (error).message
```

Comment:
I assume you added this comment because you're using a toolchain that complained about a posssible type error on this line. The problem is that unless you can convince the rest of the project to use the same tool, or to include a type checking step in the linting process, someone might just come along and remove the comment because they think that it hurts the readability of the code.

Reply:
I personally don't like this kind of thing at all. I just don't find it readable or helpful.